integration-tests/src/test/java/tachyon/master/JournalIntegrationTest.java

```diff
@@ -616,4 +620,71 @@ private void rawTableTestUtil(FileInfo fileInfo) throws Exception {
 
     fsMaster.stop();
   }
+
+  @Test
+  @LocalTachyonClusterResource.Config(tachyonConfParams = {
+      Constants.SECURITY_AUTHENTICATION_TYPE, "SIMPLE",
+      Constants.SECURITY_AUTHORIZATION_PERMISSION_ENABLED, "true",
+      Constants.SECURITY_GROUP_MAPPING, FakeUserGroupsMapping.FULL_CLASS_NAME})
+  public void setAclTest() throws Exception {
```

Comment:
why put this test is in `JournalIntegrationTest` rather than in `FileSystemIntegrationTest`?